docs/tenancy/1.x/identification-general.md

```diff
@@ -8,21 +8,128 @@ tags:
+# Tenant Identification
+
+- [Overview](#overview)
+- [Tenant Setup](#tenant-setup)
+  - [Tenant Contract](#tenant-contract)
+    - [Tenant Model Trait](#tenant-model-trait)
+  - [Tenant Registration](tenant-registration)
+- [Next Steps](#next-steps)
+  - [Lifecycle Hooks](#lifecycle-hooks)
+  - [Identification Drivers](#identification-drivers)
+
+## Overview
+
+The tenant identification procedure decides which tenant you are trying to service. Although using the `TenantResolver` isn't absolutely necessary, doing so offloads a great deal of the tenancy business logic to the application.
+
+The `TenantResolver` dispatches a few events used by the identification drivers to identify the currently requested [Tenant](what-is-a-tenant). By using an event you are not limited to only one driver. Each driver will only trigger it's own contract when it is identifying. However, if identification is triggered without a specific driver, it will try to identify all drivers. The first driver to respond with a valid tenant object will cause the other drivers to be ignored.
+
+> Before making any changes to any code, it is recommended that you read though this entire document first.
+
+## Tenant Setup
+
+### Tenant Contract
+
+The tenant contract `Tenancy\Identification\Contracts\Tenant` marks a specific 
+model as a valid tenant. Have the model implement the contract methods to get started:
+
+```php
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+use Tenancy\Identification\Contracts\Tenant;
+
+class User extends Model implements Tenant
+{
+    /**
+     * The attribute of the Model to use for the key.
+     *
+     * @return string
+     */
+    public function getTenantKeyName(): string
+    {
+        return 'id';
+    }
+
+    /**
+     * The actual value of the key for the tenant Model.
+     *
+     * @return string|int
+     */
+    public function getTenantKey()
+    {
+        return $this->id;
+    }
+
+    /**
+     * A unique identifier, eg class or table to distinguish this tenant Model.
+     *
+     * @return string
+     */
+    public function getTenantIdentifier(): string
+    {
+        return get_class($this);
+    }
+}
+```
+
+This will force your User model to implement some methods
+required for tenancy to do its work. 
+
+#### Tenant Model Trait
+
+Of course tenancy offers an easy way of applying the methods
+required by the contract to the model by using a trait for those specific functions.
+
+```php
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+use Tenancy\Identification\Concerns\AllowsTenantIdentification;
+use Tenancy\Identification\Contracts\Tenant;
+
+class User extends Model implements Tenant
+{
+    use AllowsTenantIdentification;
+}
+```
+
+Your first valid tenant model is a fact. Now we need to make sure our application
```

Comment:
Get rid of the model here

Reply:
I believe this part was a copy/paste. (got rid of model)

But do you know the intention of "Your first valid tenant is a fact" ? It sounds odd to me and could possibly use some re-work